lib/beta-flag/test/index.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import BetaFlag from '../';
+
+describe('Component', function () {
+  const wrapper = shallow(<BetaFlag />);
+  const children = wrapper.children().nodes;
+  it('should render our <BetaFlag /> component', function (done) {
+    expect(children).to.have.length(1);
+    done();
+  });
+  it('should render the correct child components', function (done) {
+    expect(wrapper.find('div')).to.have.length(1);
+    expect(wrapper.find('p')).to.have.length(1);
```

Comment:
Same comments as above really, it's hard to see what the value of this test is in terms of confirming that things are working as we expect.